restler/engine/core/driver.py

```diff
@@ -839,4 +839,5 @@ def replay_sequence_from_log(replay_log_filename, token_refresh_cmd):
         execute_token_refresh_cmd(token_refresh_cmd)
 
     # Send the requests
+    SequenceTracker.set_origin('replay')
     sequence.replay_sequence()
```

Comment:
please add a try/finally around these two lines, with 'clear_origin' in the finally

Reply:
@marina-p thanks for the quick review! My latest push addresses both comments.

Please let me know if you have a preference on how commits are managed. My latest push has each fix as its own commit, but I am happy to rebase and fixup my earlier commits if you prefer.

Reply:
Separate commits are fine for any intermediate changes.  Please squash-merge when merging the PR.